daemon/test/test.sh

```diff
@@ -5,7 +5,13 @@ cd `git rev-parse --show-toplevel`/daemon/test
 
 . scripts/vars.sh
 
-scripts/setup.sh
+# If bitcoind not already running, start it.
+if ! $CLI getinfo >/dev/null 2>&1; then
```

Comment:
So this would still shut down `bitcoind` if it was not running initially? Without `-rpcwait` we'll also want to give it some time to startup. I'll keep that in mind when writing the travis script.